frontend/ios/DroidKaigi 2019/Scenes/MainViewController.swift

```diff
@@ -78,6 +78,9 @@ extension MainViewController: DrawerViewControllerDelegate {
         case .floorMap:
             let floorMap = FloorMapViewController.instantiateFromStoryboard()
             navigationController?.pushViewController(floorMap, animated: false)
+        case .announce:
+            let announce = AnnouncementsViewController.instantiateFromStoryboard()
+            navigationController?.pushViewController(announce, animated: false)
```

Comment:
I changed my implementation following the change of moving menus to navigation drawer 🙋

Reply:
I will delete unused icon image🐈